background/handle-temporary.js

```diff
@@ -0,0 +1,5 @@
+chrome.runtime.onMessage.addListener((request, sender, sendResponse) => {
+  if (!request.setTemporaryState) return;
+  scratchAddons.globalState.temporary[request.key] = request.value;
+  sendResponse();
+});
```

Comment:
why is this necessary

Reply:
If I just put the key in the url, any origin could redirect and put in the code. This is needed to restrict it to fluffyscratch.